connector/connect/client/jvm/src/main/scala/org/apache/spark/sql/connect/client/SparkConnectClient.scala

```diff
@@ -463,6 +464,14 @@ object SparkConnectClient {
       this
     }
 
+    def sessionId(value: String): Builder = {
+      require(value != null)
```

Comment:
should we check that it's UUID here, to error out early, before server errors this out on first request?

Reply:
in that case throw INVALID_HANDLE.FORMAT error-class.